src/views/Buckets/Bucket-Files/Bucket-Details.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div class="row mb-1">
+    <div class="col col-sm-12">
+      <b-card no-body>
+        <b-card-header>
+          <h5 class="row mb-4">
+            <span class="col-sm-6">
+              Bucket ID: {{ bucket.id }}
+            </span>
+            <span class="col-sm-6">
+              Status: {{ bucket.status }}
+            </span>
+          </h5>
+        </b-card-header>
+        <b-card-body>
+          <div class="row">
+            <dl class="col-md-6 mb-0">
+              <span class="row">
+                <dt class="col-sm-6">
+                  Storage:
+                </dt>
+                <dd class="col-sm-6">
+                  {{ bucket.storage }} GBMo.
+                </dd>
+              </span>
+
+              <span class="row">
+                <dt class="col-sm-6">
+                  Public Permissions:
```

Comment:
This isn't implemented and cruft on the model

Reply:
Proposal on how such implementation can exist https://github.com/Storj/sips/pull/41/files